core/controllers/creator_dashboard.py

```diff
@@ -264,20 +248,20 @@ class NotificationsHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    @acl_decorators.can_access_creator_dashboard
```

Comment:
It's not called when user is logged out.

It is called in top navigation bar [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js#L87)